src/content/docs/en/getting-started.mdx

```diff
@@ -102,6 +103,12 @@ Join us in the [Astro Discord](https://astro.build/chat/) to share with and get
 
 [Astro changelog](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md)
 
+<TranslatorList githubRepo="withastro/docs">
+## Translate
+
+These docs translation are brought to you by all these amazing people. [Help us with translations!](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md)
+</TranslatorList>
```

Comment:
Nit, but I don’t think we even need to use `<slot />` here. This would work too right?

```suggestion
## Translate

These docs translation are brought to you by all these amazing people. [Help us with translations!](https://github.com/withastro/docs/blob/main/contributor-guides/translating-astro-docs.md)

<TranslatorList githubRepo="withastro/docs" />
```

Reply:
Ah, I guess you’re relying on the slot to hide the section on `/en/`. Except we get a broken link because the heading still shows up in the table of contents. We can avoid that with an `<h2>` but should maybe add a comment that translations should use a `##` Markdown heading.

Reply:
The stats seem to be correct, in Portuguese specifically, you and Fred get attributed 3 contributions because of commits setting/changing the internationalization setup within Astro, which will propagate to all languages. Same with people who did commits fixing all languages of small typos, etc. The current data looks like this:

```js
[
  { id: 61414485, login: 'Yan-Thomas', contributions: 80 },
  { id: 357379, login: 'delucis', contributions: 3 },
  { id: 622227, login: 'FredKSchott', contributions: 3 },
  { id: 19380403, login: 'ralacerda', contributions: 3 },
  { id: 71955135, login: 'favanso', contributions: 3 },
  { id: 3019731, login: 'Princesseuh', contributions: 2 },
  { id: 71379045, login: 'andremralves', contributions: 2 },
  { id: 62016742, login: 'Jvictorvieira', contributions: 2 },
  { id: 16581093, login: 'vitoriapena', contributions: 2 },
  { id: 37726261, login: 'wtchnm', contributions: 2 },
  { id: 16712703, login: 'JuanM04', contributions: 2 },
  { id: 95082903, login: 'grace-anand', contributions: 1 },
  { id: 90774048, login: 'IgStefano', contributions: 1 },
  { id: 64310361, login: 'TheOtterlord', contributions: 1 },
  { id: 69775937, login: 'pedroarthuralvesdeoliveira', contributions: 1 },
  { id: 13081219, login: 'Dcerverizzo', contributions: 1 },
  { id: 5098874, login: 'sarah11918', contributions: 1 },
  { id: 5558193, login: 'mhstrkmp', contributions: 1 },
  { id: 7889778, login: 'morinokami', contributions: 1 },
  { id: 34116392, login: 'bluwy', contributions: 1 },
  { id: 299173, login: 'lukemcdonald', contributions: 1 },
  { id: 24359130, login: 'swift502', contributions: 1 },
  { id: 46300090, login: 'rishi-raj-jain', contributions: 1 },
  { id: 31109854, login: 'viniciusflv', contributions: 1 },
  { id: 46791833, login: 'kevinzunigacuellar', contributions: 1 }
]
```

Many contributors have done more reviews than translations, so that's why it doesn't look accurate.

Reply:
Ah, I bet it was because I was thinking of our old contributor to Portuguese who deleted their GitHub account. They show up on Astro Badges too: https://astro.badg.es/v1/contributor/ghost/ 👻